Tests/ParameterSpaces/parameter_space_mock.cpp

```diff
@@ -419,8 +419,6 @@ void A2dParameterSpaceMock::NurbsBookExe3_8() {
   EXPECT_CALL(*this, InsertKnotMock(kDimension1, kParametricCoordinate0_5, kMultiplicity2, IsGe0_0AndLt0_5))
       .WillRepeatedly(DoAll(InvokeWithoutArgs(this, &A2dParameterSpaceMock::NurbsBookExe3_8Subdivided),
                             Return(kInsertionInformationSubdivide)));
```

Comment:
I just noticed the typo in `kInsertionInformationSubdivide`. Could you please fix it to `kInsertionInformationSubdivided`?